src/test/java/org/wso2/carbon/connector/integration/test/cerner/CernerConnectorIntegrationTest.java

```diff
@@ -0,0 +1,396 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.connector.integration.test.cerner;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+/**
+ * Integration test
+ */
+public class CernerConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+
+    private Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+    private Map<String, String> apiRequestHeadersMap = new HashMap<String, String>();
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+                System.getProperty("connector_version") + ".zip";
+        init(connectorName);
+        eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Content-Type", "application/json+fhir");
+        eiRequestHeadersMap.put("Accept", "application/json+fhir");
+        apiRequestHeadersMap.putAll(eiRequestHeadersMap);
+    }
+
+    /**
+     * Positive test case for getConformance method with mandatory parameters.
+     */
+    @Test(enabled = true, description = "cerner {getConformance} integration test with mandatory parameters.")
+    public void getConformanceWithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:getConformance");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "getConformance.json");
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/metadata";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
```

Comment:
Can we assert more properties from response body ? fix other test cases as well.

refer [1]. 
[1] https://github.com/wso2-extensions/esb-connector-salesforcerest/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/salesforcerest/SalesforceRestConnectorIntegrationTest.java#L118